README.md

```diff
@@ -5,11 +5,13 @@ This repository contains details of this AMWA Specification, including Node, Reg
 ## Getting started
 
 Readers are advised to be familiar with:
-* The JT-NM Reference Architecture (http://jt-nm.org/)
+* The JT-NM Reference Architecture <http://jt-nm.org/>
```

Comment:
We should make this consistent across the specs: https://github.com/search?q=org%3AAMWA-TV+JT-NM+Reference+Architecture&type=Code
I'm not sure why we don't use the **nmos-audio-channel-mapping** style everywhere.
```suggestion
* The [JT-NM Reference Architecture](http://jt-nm.org/RA-1.0/)
```